spec/stepmod/concept_spec.rb

```diff
@@ -96,10 +100,6 @@
         that indicates set S2 consists of all members of U that are not members of S1.
 
 
-        NOTE: This term is incompletely defined in this document.
```

Comment:
We still need to keep this NOTE for items in Clause 4 (`04x-*`) and Clause 5 (`05x-*.adoc`).

Reply:
> 3. As every Clause 4/5 term entry has a source, remove the NOTE, and add the “incompletely defined in this document” sentence to Clause 4 and 5 at the top.

@ronaldtse this is mentioned in the [ticket](https://github.com/metanorma/stepmod-utils/issues/123), are we not doing this right now or am I missing something here?

Reply:
Yes you are correct. I got confused. Instead please ensure we have the “[.source]” entry for the Clause 4 and Clause 5. Thanks.